.travis/travis_utils.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+build_message(){
+    # $1 = build message
+    echo
+    echo =========BUILD MESSAGE=========
+    echo "$@"
+    echo ===============================
+    echo
+}
+
+login_docker(){
+  docker login --username=$DOCKER_USER --password=$DOCKER_PASS
+}
+
+prepare_ci(){
+  DOCKER_ORG=treehouses
+  DOCKER_REPO=kolibri-tags
+  BRANCH=$TRAVIS_BRANCH
+  COMMIT=${TRAVIS_COMMIT::8}
+}
+
+push_a_docker(){
+  build_message pushing $1
+	docker push $1
+	build_message done pushing $1
+}
+
+tag_a_docker(){
+  build_message processing $2
+	docker tag $1 $2
+	build_message done processing $2
+}
+
+prepare_kolibri_amd64(){
+  build_message prepare Kolibri amd64 docker...
+  KOLIBRI_AMD64=$DOCKER_ORG/$DOCKER_REPO:amd64-$BRANCH-$COMMIT
+  KOLIBRI_AMD64_LATEST=$DOCKER_ORG/$DOCKER_REPO:amd64-latest
+}
+
+prepare_kolibri_arm(){
+  build_message prepare Kolibri arm docker...
+  KOLIBRI_ARM=$DOCKER_ORG/$DOCKER_REPO:arm-$BRANCH-$COMMIT
+  KOLIBRI_ARM_LATEST=$DOCKER_ORG/$DOCKER_REPO:arm-latest
+}
+
+
+prepare_kolibri_aarch64(){
```

Comment:
can we call this `arm64` and not `aarch64`

Reply:
Linux officially calls it `aarch64` so I named it that way. Also for someone new, a quick look at the code, an oversight might be caused when someone sees `arm64` and thinks that its `amd64` as both appear pretty similar. (Also happened to me  :grin: )

But if you still want to, I can change that.

Reply:
:D

I want cause your officially is only because of copyright apple things ...
... so now same CPU in GNU now has 2 names

an normal one owned by apple for apple software
and a free totally strange one because of copyright
as a real pirate I do care ... not of copyrights  :P

Reply:
sure, np, i'll change it to `arm64`